dockertest/environment.py

```diff
@@ -62,3 +62,24 @@ def get_selinux_context(path):
     """
     # First list item is null-terminated string length
     return selinux.getfilecon(path)[1]
+
+
+def selinux_is_enforcing():
+    """
+    Return True if selinux is in enforcing mode.
+    """
+    # These constants are undefined in selinux module
+    # pylint: disable=C0103
+    ERROR = -1
+    OLDKERNEL = 0
+    ENFORCING = 1  # Travis complains about missing selinux.ENFORCING
+    try:
+        if selinux.selinux_status_open(OLDKERNEL) == ERROR:
+            raise ValueError("Could not r/o mmap /sys/fs/selinux/status")
+        mode = selinux.selinux_status_getenforce()
+        if mode == ERROR:
+            raise ValueError("Error checking SELinux is disabled,"
+                             " permissive, or enforcing")
+    finally:
+        selinux.selinux_status_close()
+    return mode == ENFORCING
```

Comment:
Where is this selinux module documented? In particular, where are these magic ERROR and OLDKERNEL and ENFORCING values described? `pydoc selinux` does not help in any way. I think future maintainers might greatly appreciate it if you just invoke `getenforce`.